__tests__/endpoints.test.js

```diff
@@ -419,3 +419,56 @@ describe("GET /api/users", () => {
       });
   });
 });
+
+describe("GET /api/articles, for sort_by query", () => {
+  test("?sort_by= responds with array of articles ordered by the given sort_by query column, defaults to the created_at date", () => {
+    return request(app)
+      .get("/api/articles?sort_by=created_at")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles.length).toBe(13);
+        expect(body.articles).toBeSortedBy("created_at", { descending: true });
+      });
+  });
+
+  test('400: responds with "invalid query" error message when given an invalid sort_by query', () => {
+    return request(app)
+      .get("/api/articles?sort_by=invalid-query")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("invalid query");
+      });
+  });
+});
+
+describe("GET /api/articles, for order query", () => {
+  test("?order=desc responds with array of articles ordered by descending order of created_at date", () => {
+    return request(app)
+      .get("/api/articles?order=desc")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles.length).toBe(13);
+        expect(body.articles).toBeSortedBy("created_at", { descending: true });
+      });
+  });
+
+  test("?order=asc responds with array of articles ordered by ascending order of created_at date", () => {
+    return request(app)
+      .get("/api/articles?order=asc")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles.length).toBe(13);
+        expect(body.articles).toBeSortedBy("created_at", { descending: false });
+      });
+  });
+
+  test('400: responds with "invalid order" error message when given an invalid order query', () => {
+    return request(app)
+      .get("/api/articles?order=invalid-order")
+      .expect(400)
+      .then(({ body }) => {
+        console.log(body);
+        expect(body.msg).toBe("invalid query");
+      });
+  });
```

Comment:
I would add another test that when you give it a sort_by query you get the correct response